test/integration/fabric-ca-certificate-service-tests.js

```diff
@@ -29,8 +29,8 @@ test('\n\n ** FabricCAServices - CertificateService Test **\n\n', async (t) => {
 		FabricCAServices.getConfigSetting('crypto-keysize', '256'); // force for npm test
 		FabricCAServices.setConfigSetting('crypto-hash-algo', 'SHA2'); // force for npm test
 
-		const caService1 = new FabricCAServices(fabricCAEndpoint1, tlsOptions, ORGS[userOrg1].ca.name);
-		const caService2 = new FabricCAServices(fabricCAEndpoint2, tlsOptions, ORGS[userOrg2].ca.name);
+		const caService1 = new FabricCAServices(fabricCAEndpoint1, tlsOptions, ORGS[userOrg1].ca.name, newDefaultCryptoSuite());
```

Comment:
Same as a previous comment but the integration tests should generally be what an application developer might write and it worries me that the application code has to change to avoid failure when migrating to a version of the SDK that includes this change